examples/lighting-app/efr32/BUILD.gn

```diff
@@ -99,26 +158,48 @@ efr32_executable("lighting_app") {
     "${chip_root}/examples/lighting-app/lighting-common",
     "${chip_root}/src/lib",
     "${chip_root}/src/setup_payload",
-    "${chip_root}/third_party/openthread/platforms:libopenthread-platform",
-    "${chip_root}/third_party/openthread/platforms:libopenthread-platform-utils",
-    "${examples_plat_dir}:efr-matter-shell",
   ]
-
-  if (chip_openthread_ftd) {
+  if (!use_rs911x && !use_wf200) {
     deps += [
+      "${examples_plat_dir}:efr-matter-shell",
+      "${chip_root}/third_party/openthread/platforms:libopenthread-platform",
+      "${chip_root}/third_party/openthread/platforms:libopenthread-platform-utils",
+    ]
+    if (chip_openthread_ftd) {
+     deps += [
       "${chip_root}/third_party/openthread/repo:libopenthread-cli-ftd",
       "${chip_root}/third_party/openthread/repo:libopenthread-ftd",
-    ]
-  } else {
-    deps += [
+     ]
+    } else {
+     deps += [
       "${chip_root}/third_party/openthread/repo:libopenthread-cli-mtd",
       "${chip_root}/third_party/openthread/repo:libopenthread-mtd",
-    ]
+     ]
+    }
+  }
+  if (use_rs911x) {
+     sources += rs911x_src_plat
+     # All the stuff from wiseconnect
+     sources += rs911x_src_sapi
+     # Apparently - the rsi library needs this SH%T (though we may not use use it)
```

Comment:
consider deleting the SH%T and express frustration with some TODO on cleaning this dependency if possible

Reply:
Resolved - removed the offensive language from BUILD.gn